src/main/webapp/app/course/manage/course-for-archive-dto.ts

```
@@ -0,0 +1,9 @@
+export class CourseForArchiveDTO {
+    id: number;
+    title: string;
+    semester: string;
+    color: string;
+    icon: string;
+
+    constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)